Source/WebCore/rendering/RenderObject.h

```diff
@@ -1128,74 +1152,19 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
             IsStickilyPositioned = 3
         };
 
-    public:
-        RenderObjectBitfields(const Node& node)
-            : m_hasRareData(false)
-            , m_beingDestroyed(false)
-            , m_needsLayout(false)
-            , m_needsPositionedMovementLayout(false)
-            , m_normalChildNeedsLayout(false)
-            , m_posChildNeedsLayout(false)
-            , m_needsSimplifiedNormalFlowLayout(false)
-            , m_preferredLogicalWidthsDirty(false)
-            , m_floating(false)
-            , m_isAnonymous(node.isDocumentNode())
-            , m_isRenderText(false)
-            , m_isRenderBox(false)
-            , m_isInline(true)
```

Comment:
This defaults to true. I'm flipping the flag to be IsBlock instead so that the default value can be 0.